Source/DataSources/KmlDataSource.js

```diff
@@ -255,6 +255,23 @@ define([
         return deferred.promise;
     }
 
+    function insertNamespaces(text) {
+        var namespaceMap = {
```

Comment:
I think it would be cleaner to have the following then build the search strings in the loop.
```
var namespaceMap = {
  xsi : 'http://www.w3.org/2001/XMLSchema-instance'
};
```